test/SmokeTests/OOProcSmokeTests/DotNetIsolated/HelloCitiesUntyped.cs

```diff
@@ -38,18 +39,19 @@ internal static class HelloSequenceUntyped
     /// <summary>
     /// Orchestrator function that calls the <see cref="SayHelloUntyped"/> activity function several times consecutively.
     /// </summary>
-    /// <param name="state">The opaque orchestration state that gets passed to the function. Code should never attempt to read this state.</param>
+    /// <param name="requestState">The serialized orchestration state that gets passed to the function.</param>
     /// <returns>Returns an opaque output string with instructions about what actions to persist into the orchestration history.</returns>
     [Function(nameof(HelloCitiesUntyped))]
-    public static string HelloCitiesUntyped([OrchestrationTrigger] string state) =>
-        DurableOrchestrator.LoadAndRun<string, string>(state, async (context, _) =>
+    public static string HelloCitiesUntyped([OrchestrationTrigger] string requestState, FunctionContext functionContext) =>
+        OrchestrationRunner.LoadAndRun<string, string>(requestState, async (context, _) =>
```

Comment:
Note that `OrchestrationRunner` is now defined in the Microsoft.DurableTask nuget package. This was possible now that we've moved away from the Functions-specific JSON protocol to the Durable Task-specific protobuf protocol.

Reply:
How does changing protocols enable this? I'm not entirely clear on that

Reply:
It wouldn't have made sense to move this code to the generic SDK because it relied on Functions-specific serialization logic (I consider the JSON serialization to be Functions-specific because the Durable Functions extension is the only place where it's used). By ditching the JSON serialization protocol and adopting the generic SDK's protobuf protocol, we no longer had any Functions-specific JSON protocols being used, even in the Functions case. Thus, I could easily consolidate more logic into the generic SDK.